### PR TITLE
Use UTC for recordings

### PR DESCRIPTION
**Pull request by NickM-27**
The nginx vod module makes query parameters very difficult so I added a path param instead. Could likely be reworked when removing the vod module.

### CODE REVIEW

**Review by blakeblackshear — CHANGES_REQUESTED** [review]

File: web/src/routes/Recording.jsx [hunk 1]
```diff
@@ -126,6 +132,7 @@ export default function Recording({ camera, date, hour = '00', minute = '00', se
   return (
     <div className="space-y-4 p-2 px-4">
       <Heading>{camera.replaceAll('_', ' ')} Recordings</Heading>
+      <div className="text-xs">Dates & Times are displayed. in local time</div>
```
[the remark this answers is not in the document]
Comment (NickM-27) [reply web/src/routes/Recording.jsx:135]: Yeah, I like that better

**Review by blakeblackshear — APPROVED** [review]

**Comment by blakeblackshear** [comment]
The changes to the folder structure will be a breaking change for viewing recordings in the media browser integration in HA. On the bright side, we should be able to simplify that code because it still relies on parsing the directory structure and we can now provide APIs given the recordings info is in the database.

**Comment by singingtelegram** [comment]
I'm receiving a pytz.exceptions.UnknownTimeZoneError error since the code replaced the underscore in the actual time zone (America/Los_Angeles) with a slash.

```
frigate    | 2022-12-29 20:44:48.241651153  [2022-12-29 20:44:48] frigate.http                   ERROR   : Exception on /vod/2022-mm/dd/hh/name/America_Los_Angeles [GET]
frigate    | 2022-12-29 20:44:48.241652946  Traceback (most recent call last):
frigate    | 2022-12-29 20:44:48.241654058    File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 2525, in wsgi_app
frigate    | 2022-12-29 20:44:48.241654789      response = self.full_dispatch_request()
frigate    | 2022-12-29 20:44:48.241655531    File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 1822, in full_dispatch_request
frigate    | 2022-12-29 20:44:48.241656122      rv = self.handle_user_exception(e)
frigate    | 2022-12-29 20:44:48.241656863    File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 1820, in full_dispatch_request
frigate    | 2022-12-29 20:44:48.241657404      rv = self.dispatch_request()
frigate    | 2022-12-29 20:44:48.241658136    File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 1796, in dispatch_request
frigate    | 2022-12-29 20:44:48.241658817      return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
frigate    | 2022-12-29 20:44:48.241659468    File "/opt/frigate/frigate/http.py", line 1101, in vod_hour
frigate    | 2022-12-29 20:44:48.241660190      int(parts[0]), int(parts[1]), int(day), int(hour), tzinfo=pytz.timezone(tz_name)
frigate    | 2022-12-29 20:44:48.241660891    File "/usr/local/lib/python3.9/dist-packages/pytz/__init__.py", line 188, in timezone
frigate    | 2022-12-29 20:44:48.241661472      raise UnknownTimeZoneError(zone)
frigate    | 2022-12-29 20:44:48.241662073  pytz.exceptions.UnknownTimeZoneError: 'America/Los/Angeles'

**Comment by NickM-27** [comment]
Thanks, I didn't consider that which was short-sighted. I'll fix that tomorrow

**Pull request merged**